packages/app/cypress/e2e/studio/studio.cy.ts

```diff
@@ -56,7 +56,6 @@ describe('studio functionality', () => {
 
       cy.window().then((win) => {
         expect(win.Cypress.config('isDefaultProtocolEnabled')).to.be.false
-        expect(win.Cypress.config('isStudioProtocolEnabled')).to.be.true
```

Comment:
Do we need another test that verifies `isStudioProtocolEnabled`?

Reply:
we don't set `isStudioProtocolEnabled` on the window anymore